protocols/IFIN/README_OT1.md

```diff
@@ -0,0 +1,39 @@
+# Enzyme Activity
+
+### Author
+[Opentrons](http://www.opentrons.com/)
+
+## Categories
+* Basic Pipetting
+    * Plate Filling
+
+## Description
+This protocol allows the robot to transfer aqueous solutions from a reservoir (plate A, 96-well plate) to an experimental plate (plate B, 96-well plate).
+
+Steps:
+1. Transfers 35 uL from plate A row 10 to plate B rows 1-4, plate A row 11 to plate B rows 5-8, and plate A row 12 to plate B rows 9-12, using the same pipette tips.
```

Comment:
Move steps to 'process'. More detailed description about enzyme activity maybe? See if trello card has any reagents added to it.